zerver/webhooks/zapier/doc.html

```diff
@@ -40,3 +40,33 @@
 <p>You're done! Example message:</p>
 
 <img class="screenshot" src="/static/images/integrations/zapier/005.png"/>
+
+<p>
+    <b>Weather Update with Zapier!</b><br />
+</p>
+<p>
+    Go to the following link <code>https://zapier.com/app/explore</code> after you sign into your zapier account<br />
+    Select Send Weather info to your app via webhooks.
```

Comment:
Use the same phrasing and capitalization that the user is going to see, and surround it with quote marks. Here, based on the screenshot, I think it should be:

> Select "Send weather info to your app via webhooks".